packages/nextjs/utils/getFiltersCount.ts

```diff
@@ -0,0 +1,26 @@
+import { useRouter } from "next/router";
+import { ParsedUrlQuery } from "querystring";
+import type { FilterGroup } from "./filters";
+import { getFilterGroups } from "./filters";
+
+export const getFiltersCount = (query: ParsedUrlQuery) => {
+  const filters = getFilterGroups();
+
+  const total = filters.reduce((acc: number, { label, value }: FilterGroup) => {
+    const selectedFilters = query[value];
+    const elements = selectedFilters?.length ?? 0;
+
+    // if a single element is selected, type would be a string instead of an array.
+    const totalFilters = typeof selectedFilters === "string" ? (elements > 0 ? 1 : 0) : elements;
+
+    return (acc = acc + totalFilters);
+  }, 0);
+
+  return total;
+}
```

Comment:
There's a missing semicolon here, and some other eslint warnings in this page. If you use VSCode, you might consider adding the following to `.vscode/settings.json` (which I see is checked into the repo 😓 ):
```json
"editor.codeActionsOnSave": {
  "source.fixAll": true
}
```